packages/orchestration/src/internal.ts

```diff
@@ -1,4 +1,30 @@
-export * from './orchestration-client.js';
+export * from './client/api/schema/index.js';
 export * from './util/index.js';
-export * from './orchestration-types.js';
+export * from './orchestration-client.js';
 export * from './orchestration-response.js';
+export * from './orchestration-stream-chunk-response.js';
+export * from './orchestration-stream-response.js';
+export * from './orchestration-stream.js';
+export type {
```

Comment:
[req] Why not `export * from './orchestration-types.js';`

Reply:
We cannot as TS will complain about the ambiguity of having same type name from generated files and `orchestration-types.js`.